src/web-server.js

```diff
@@ -1,5 +1,6 @@
 //Setup
 "use strict";
+require("dotenv").config();
```

Comment:
Not sure if we should add this. Most people use import-map-deployer via Docker, where environment variables are set in the dockerfile. Example at https://github.com/joeldenning/live-import-map-deployer/blob/df3f5ea7c51286e93ee04309c11cb82baf57c554/Dockerfile#L3. Additionally, most configuration is set via the [configuration file](https://github.com/single-spa/import-map-deployer#configuration-file) which is not the `.env` file.

For those reasons, I lean towards removing dotenv. @frehner do you have thoughts?

Reply:
After thinking this through some more, I think it's fine. Doesn't hurt when people don't want it.